docs/api/en/lights/LightProbe.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<base href="../../../" />
+		<script src="list.js"></script>
+		<script src="page.js"></script>
+		<link type="text/css" rel="stylesheet" href="page.css" />
+	</head>
+	<body>
+		[page:Object3D] &rarr; [page:Light] &rarr;
+
+		<h1>[name]</h1>
+
+		<p class="desc">
+			Light probes are an alternative way of adding light to a 3D scene. Unlike classical light sources (e.g. directional,
+			point or spot lights), light probes do not emit light. Instead they store information about light passing through
+			3D space. During rendering, the light that hits a 3D object is approximated by using the data from the light probe.
+		</p>
+
+		<p class="desc">
+			Light probes are usually created from (radiance) environment maps. The class [page:LightProbeGenerator] can
+			be used to create light probes from instances of [page:CubeTexture] or [page:WebGLCubeRenderTarget].
+			However, light estimation data could also be provided in other forms e.g. by WebXR. This enables the rendering
+			of augmented reality content that reacts to real world lighting.
+		</p>
+
+		<p class="desc">
+			The current probe implementation in three.js only supports so called diffuse light probes. This type of light probe
+			is a different representation of an irradiance environment map. Other types of light probes like reflection probes
+			are not yet supported.
```

Comment:
@Mugen87 Thank you for adding this documentation!

>reflection probes are not yet supported.

A three.js environment map is a reflection probe. Right?

>Other types of light probes ... are not yet supported.

What are the "other types of light probes" in your opinion?

>The current probe implementation in three.js only supports so called diffuse light probes.

I think that is all it will support. I have considered changing the name to `IrradianceProbe`.

Reply:
The thing is that other engines have entities called `ReflectionProbe`. Hence, I wanted to highlight that such a class does not exist in `three.js`. If you think this can be formulated in a better way, feel free to clarify this section.

> I think that is all it will support. I have considered changing the name to IrradianceProbe.

Although calling the class `IrradianceProbe` is correct from an academical point of view, I fear users will be confused what the term "irradiance" actually means. In fact, most users struggle with basic PBR terms like direct/indirect light or radiance/irradiance. Maybe it's better to keep the class name simple...

Reply:
OK. I'll do something... I usually opt for saying less.